packages/manager/src/features/Longview/LongviewLanding/LongviewClients.tsx

```diff
@@ -178,8 +177,7 @@ export const LongviewClients = (props: LongviewClientsCombinedProps) => {
   // If this value is defined they're not on the free plan
   // and don't need to be CTA'd to upgrade.
 
-  const isLongviewPro = !isEmpty(activeSubscription);
-
+  const isLongviewPro = !!Object.keys(activeSubscription).length;
```

Comment:
Can we make this (and other instances like this) more explicit?
```suggestion
  const isLongviewPro = Object.keys(activeSubscription).length > 0;
```

Reply:
Addressed in 582beefd0772785f7fc5c313c05f14cfb2918d70